includes/bootstrap.php

```diff
@@ -54,6 +54,7 @@ function() {
 	'rest_post_dispatch',
 	function( $response ) {
 		$response->header( 'X-Distributor', 'yes' );
+		$response->header( 'X-Distributor-Version', DT_VERSION );
```

Comment:
@peterwilsoncc I am not aware why we add distributor-related information to all API requests. Is it required? Can we limit it to distributor-specific rest API requests?

Reply:
> I am not aware why we add distributor-related information to all API requests.

Not sure of the design decision here, let's open a ticket as a maybe later.